site2/website/blog/2017-09-26-adding-rss.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Adding RSS Support
+author: Eric Nakagawa
```

Comment:
probably we can't commit this name, please change it to "Foo bar" or anything else

Reply:
Yes, I have to change a whole lot more for Blog posts. Going to copy-paste your HerdDb blogs from LinkedIn and @sijie 's blogs from StreamNative into here. This will be taken out in the next commit.